modules/platform/nuxeo-platform-convert/src/test/java/org/nuxeo/ecm/platform/convert/tests/TestCommandLineConverter.java

```diff
@@ -56,6 +58,11 @@ public void testCommandLineConverter() throws Exception {
 
         Blob mainBlob = result.getBlob();
         assertEquals("hello.png", mainBlob.getFilename());
+
+        // NXP-29067: make sure the parent path is prefixed with the command line name
+        FileBlob fileBlob = ((FileBlob) mainBlob);
+        String parentPath = fileBlob.getFile().getParent();
```

Comment:
Actually `getFile()` is part of the `Blob` interface so there's no need for a cast at all — unless you want to verify that it's actually a `FileBlob`.